README.md

```diff
@@ -23,7 +23,7 @@ will also take care of checksum verification of the downloaded files and retry f
 
 ## Compatibility
 
-FastqHeat needs Python 3.7 or newer.
+FastqHeat needs Python 3.9.
```

Comment:
Maybe something like "FastqHeat are being developed and tested under Python 3.9.x". What do you think?

Reply:
I think this could work as well.

Reply:
This isn't the wording that was proposed in the linked task, by the way.

Reply:
My point is:
> FastqHeat needs Python 3.9.

states that the package definitely will not work on other versions of Python rather than 3.9, so a user has to install Python 3.9 before even trying to execute our package - which for some people may be a hurdle 

> FastqHeat is being developed and tested under Python 3.9.x.

states that a user is free to try on whatever version of Python he/she has or willing to, but he/she has been warned that it may or may not work on other versions of Python, or may work improperly

Everything is just my opinion, though